hs_tools_resource/models.py

```diff
@@ -107,6 +103,63 @@ def remove(cls, element_id):
         raise ValidationError("SupportedResTypes element can't be deleted.")
 
 
+class SupportedSharingStatusChoices(models.Model):
+    description = models.CharField(max_length=128)
+
+    def __unicode__(self):
+        return self.description
+
+
+class SupportedSharingStatus(AbstractMetaDataElement):
+    term = 'SupportedSharingStatus'
+    sharing_status = models.ManyToManyField(SupportedSharingStatusChoices, null=True, blank=True)
+
+    def get_sharing_status_str(self):
+        return ', '.join([parameter.description for parameter in self.sharing_status.all()])
+
+    @classmethod
+    def create(cls, **kwargs):
+        if 'sharing_status' in kwargs:
+            metadata_obj = kwargs['content_object']
+            new_meta_instance = SupportedSharingStatus.objects.create(content_object=metadata_obj)
+            for sharing_status in kwargs['sharing_status']:
+                qs = SupportedSharingStatusChoices.objects.filter(description__iexact=sharing_status)
+                if qs.exists():
+                    new_meta_instance.sharing_status.add(qs[0])
+                else:
+                    new_meta_instance.sharing_status.create(description=sharing_status)
+            return new_meta_instance
+        else:
+            raise ObjectDoesNotExist("No sharing_status parameter was found in the **kwargs list")
+
+    @classmethod
+    def update(cls, element_id, **kwargs):
+        meta_instance = SupportedSharingStatus.objects.get(id=element_id)
+        if meta_instance:
```

Comment:
No need for this ```if```. The previous statement will throw an exception if the object is not found.